config.json

```diff
@@ -572,7 +572,13 @@
       "core": false,
       "difficulty": 7,
       "slug": "word-search",
-      "topics": null,
+      "topics": [
+        "string",
+        "words",
+        "loops",
+        "matrices",
+        "matching"
```

Comment:
This is also not a topic in the topics list. Maybe `searching` would be better? I would also add `games` to this list since `word-search` is a game :)

Reply:
hmm okay I will make the changes you want. ;)

Reply:
So you mean we have to pick a topic from the TOPICS.txt

Reply:
The list of topics (TOPICS.txt) is there to make sure the topics that are shared are consistent. This is to avoid the case where for example one track uses `string` as a topic and a different track uses `strings`. So when adding a topic you should first check the list to see if the topic you want (or something very similar) is already there. If so you should add the version in the list to be consistent. If the topic you want is not there then there are a few options:
- If you think it's a valid topic that could be used in several language tracks then you can open a PR to update the topics list.
- If you think it's a valid topic that is specific to this language track then it's fine to add it without it being in the topics list.

Does that make sense? :)

Reply:
Yep , now it makes sense :p hehe